comms/models.py

```diff
@@ -7,11 +8,30 @@
 from twilio.rest import Client
 
 from account.models import Parent, Instructor
+from course.models import Course
+
 
 sg = SendGridAPIClient(api_key=settings.SENDGRID_API_KEY)
 twilio = Client(settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN)
 
 
+class Announcement(models.Model):
+    subject = models.TextField()
+    body = models.TextField()
+    course = models.ForeignKey(
+        Course,
+        on_delete=models.PROTECT
+    )
+    poster = models.ForeignKey(
+        get_user_model(),
```

Comment:
If only instructors can post, perhaps make instructors the foreign key instead of user?

Reply:
I'm allowing instructors, admins, and receptionist to post that's why I used the get_user_model method.